src/cljs/orcpub/dnd/e5/spell_subs.cljs

```diff
@@ -854,7 +860,7 @@
         tiefling-option-cfg]))))))
 
 
-(defn base-class-options [spell-lists spells-map plugin-subclasses-map language-map weapons-map invocations]
+(defn base-class-options [spell-lists spells-map plugin-subclasses-map language-map weapons-map invocations boons]
```

Comment:
This doesn't have to be done now, but we should really convert all these parameters into a single map and destructure at some point. More than 3-4 params is a lot...